lib/protoparser/datadogsketches/parser.go

```diff
@@ -0,0 +1,292 @@
+package datadogsketches
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	"github.com/VictoriaMetrics/easyproto"
+)
+
+var (
+	epsillon   = 1.0 / 128
+	gamma      = 1 + 2*epsillon
+	gammaLn    = math.Log(gamma)
+	defaultMin = 0.981e-9
+	bias       = 1 - int(math.Floor(math.Log(defaultMin)/gammaLn))
+	quantiles  = []float64{0.5, 0.75, 0.9, 0.95, 0.99}
+)
+
+// Metric stores metrics extracted from sketches
+type Metric struct {
+	Name   string
+	Labels map[string]string
+	Points []float64
+}
+
+// SketchPayload stores sketches extracted from /api/beta/sketches endpoint
+//
+//	message SketchPayload {
+//		 repeated Sketch sketches = 1 [(gogoproto.nullable) = false];
```

Comment:
`[(gogoproto.nullable) = false]` has no any sense here, so let's remove it in order to reduce confusion level

```suggestion
//		 repeated Sketch sketches = 1;
```

Reply:
removed it as well